charts/steward/templates/config-pipelineruns.yaml

```diff
@@ -60,6 +83,8 @@ data:
     jenkinsfileRunner.podSecurityContext.fsGroup: "1000"
 
   networkPolicy: {{ default ( .Files.Get "data/pipelineruns-default-networkpolicy.yaml" ) .Values.pipelineRuns.networkPolicy | quote }}
+  limitRange: {{ default ( .Files.Get "data/pipelineruns-default-limitrange.yaml" ) .Values.pipelineRuns.limitRange | quote }}
+  resourceQuota: {{ default ( .Files.Get "data/pipelineruns-default-resourcequota.yaml" ) .Values.pipelineRuns.resourceQuota | quote }}
```

Comment:
This file is not in the data folder. Is this missing?

Reply:
for the limit range we added default values in the data directory but for resource quota I think we decided to add it to installer. @anbrsap Is that correct?

Reply:
Yes, we do not set a quota by default, as it is not generally applicable.

Reply:
So, this should work:
```suggestion
  resourceQuota: {{ .Values.pipelineRuns.resourceQuota | quote }}
```